components/organisms/Footer.js

```diff
@@ -0,0 +1,85 @@
+import PropTypes from "prop-types";
+
+/**
+ * footer element for all pages
+ */
+
+export function Footer(props) {
+  return (
+    <div className="w-full mb-4">
+      <div className="w-full h-auto  bg-dk-blue bg-footer-parliament-image bg-no-repeat bg-right-bottom ">
+        <div className="w-full  py-6 layout-container  ">
+          <ul className="flex flex-wrap justify-between ">
+            {" "}
+            {props.footerBoxLinks.map((value, index) => {
+              return (
+                <li
+                  key={index}
+                  className="text-white w-64 md:w-56 lg:w-80   my-2   hover:underline"
+                >
+                  <a href={value.footerBoxlink}>{value.footerBoxLinkText}</a>
+                </li>
+              );
+            })}
+          </ul>
+        </div>
+      </div>
+      <div className="w-full h-auto flex flex-col md:flex-row  items-center bg-footer-white mt-6  justify-between layout-container ">
+        <div className="w-full flex flex-col md:flex-row  justify-between mr-6  ">
+          <ul className="w-full flex flex-col md:flex-row  list-inside md:list-outside list-disc  ">
+            {" "}
+            {props.links.map((value, index) => {
+              return (
+                <li
+                  key={index}
+                  className="text-canada-ca-link-colour p-2 md:pt-4 md:pr-4  md:pb-4  w-full hover:underline "
+                >
+                  <a href={value.link}>{value.linkText}</a>
+                </li>
```

Comment:
so we can remove the padding here. 

the canada dot ca list doesn't have padding on the list element at all, rather it jumps into a 2-column layout once there is not enough space for the links and the wordmark.

<img width="1381" alt="Screen Shot 2021-04-09 at 11 02 08" src="https://user-images.githubusercontent.com/2454380/114200407-2fd00880-9923-11eb-8bb5-76862048c097.png">

Reply:
Also the list element circles look pretty good, but we want to keep them grey even when the links are clicked (you can see in the photo above this one that the bullets are grey vs ours which are the same colour as the link (purple).

<img width="499" alt="Screen Shot 2021-04-09 at 11 04 00" src="https://user-images.githubusercontent.com/2454380/114200679-70c81d00-9923-11eb-9345-ad09fd700f5c.png">